### PR TITLE
fix(favorite): #MED-239 fix fetch favorite

### PR DESCRIPTION
**Pull request by Mattox0**
## Describe your changes

Fix fetch favorite in textbooks page and resource page

## Checklist tests

- Check if favorite is displayed

## Issue ticket number and link

[MED-239](https://jira.support-ent.fr/browse/MED-239)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [X] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: frontend/src/components/list-card/ListCard.scss
Comment (Mattox0) [inline frontend/src/components/list-card/ListCard.scss:1]: pas de modif

File: frontend/src/components/list-card/ListCard.tsx
Comment (Mattox0) [inline frontend/src/components/list-card/ListCard.tsx:1]: pas de modif

File: frontend/src/components/search-card/SearchCard.scss
Comment (Mattox0) [inline frontend/src/components/search-card/SearchCard.scss:1]: pas de modif

File: frontend/src/core/const/icon-mapping.const.ts
Comment (Mattox0) [inline frontend/src/core/const/icon-mapping.const.ts:1]: pas de modif

File: frontend/src/styles/page/home.scss
Comment (Mattox0) [inline frontend/src/styles/page/home.scss:1]: pas de modif

**Review by niko-amosse — APPROVED** [review]

**Review by JinDouang — APPROVED** [review]

**Pull request merged**